src/promptflow/promptflow/_sdk/operations/_local_azure_connection_operations.py

```diff
@@ -44,12 +44,19 @@ def _client(self):
     @classmethod
     def _get_credential(cls):
         from azure.identity import DefaultAzureCredential, DeviceCodeCredential
+        from azure.ai.ml._azure_environments import _get_default_cloud_name, EndpointURLS, _get_cloud, AzureEnvironments
 
         if is_from_cli():
             try:
                 # Try getting token for cli without interactive login
-                credential = DefaultAzureCredential()
-                get_base_token(credential=credential)
+                cloud_name = _get_default_cloud_name()
+                if cloud_name != AzureEnvironments.ENV_US_GOVERNMENT:
```

Comment:
why need special compare here?

Reply:
For sovereign, it is necessary to: `authority=authority` and `exclude_shared_token_cache_credential=True`.

Reply:
When submitting the flow for local testing, it is written in the following way:
![image](https://github.com/microsoft/promptflow/assets/23182548/2b610e5a-1127-445b-abff-653ecb7b82cd)

![image](https://github.com/microsoft/promptflow/assets/23182548/7be3741b-b088-43df-bbe6-224a9d9aef8a)